manifests/d/DigitalOcean/Doctl/1.109.1/DigitalOcean.Doctl.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DigitalOcean.Doctl
+PackageVersion: 1.109.1
+PackageLocale: en-US
+Publisher: DigitalOcean
+PublisherUrl: https://www.digitalocean.com/
+PublisherSupportUrl: https://github.com/digitalocean/doctl/issues
+Author: DigitalOcean
+PackageName: doctl
+PackageUrl: https://github.com/digitalocean/doctl
+License: Apache-2.0
+LicenseUrl: https://github.com/digitalocean/doctl/blob/HEAD/LICENSE.txt
+Copyright: Copyright (c) DigitalOcean
+CopyrightUrl: https://github.com/digitalocean/doctl/blob/main/LICENSE.txt
+ShortDescription: The official command line interface for the DigitalOcean API.
+Moniker: doctl
+Tags:
+- digitalocean
+- doctl
+ReleaseNotes: |-
+  Changelog
+  - b5d4caa doctl auth list: respect json output flag. (#1547)
+
+
+ReleaseNotesUrl: https://github.com/digitalocean/doctl/releases/tag/v1.108.0
```

```suggestion
ReleaseNotes: |-
  Changelog
  - b5d4caa doctl auth list: respect json output flag. (#1547)
ReleaseNotesUrl: https://github.com/digitalocean/doctl/releases/tag/v1.109.1
```

Extra newlines should be removed, and suggested the correct ReleaseNotesUrl